docs/reference/plugins.md

```diff
@@ -1 +1,16 @@
 # Plugins
+
+Shift supports the integration of custom Plugins. Shift Plugins are Qt based UIs which can interact with Shift to display additional information, perform actions or modify the behaviour of the application.
+
+## Shift Native Plugins
+
+Shift ships with two example plugins which can be found in the *Tools* section in the top menu bar. The desired plugin can be opened by clicking on the corresponding entry.
+Once a Plugin is opened, the Shift UI can be customized with the best suited layout.
+
+The *Operators List* Plugin helps us visualize the list of operators in the workflow and shows the their current progress.

```

Comment:
I would also mention that shows the order of execution even before it gets executed.

Reply:
Hmm add maybe a note for Condition or iterators?

Reply:
done